test/js/upload-wizard/file-upload-control.test.js

```diff
@@ -5,14 +5,22 @@ import '@testing-library/jest-dom/extend-expect'
 import { StudyContext } from 'components/upload/upload-utils'
 import FileUploadControl from 'components/upload/FileUploadControl'
 import { fireFileSelectionEvent } from '../lib/file-mock-utils'
-
+import * as UserProvider from '~/providers/UserProvider'
 
 describe('file upload control defaults the name of the file', () => {
   afterEach(() => {
     // Restores all mocks back to their original value
     jest.restoreAllMocks()
   })
 
+  beforeEach(
+    jest
+      .spyOn(UserProvider, 'getFeatureFlagsWithDefaults')
+      .mockReturnValue({
+        clientside_validation: true
+      })
+  )
+
```

Comment:
This is not a blocking concern in itself, but per my blocking comment, this code would be extraneous with the more robust, default-to-CSFV conditional.

```suggestion
```

The same applies to equivalent code in other automated test changes in this PR.